tmt/export.py

```diff
@@ -212,7 +213,8 @@ def create_nitrate_case(test):
         category = 'Sanity'
 
     # Create the new test case
-    summary = test.node.get('extra-summary', test.summary)
+    summary = test.node.get('extra-summary', os.path.basename(test.node.root)
```

Comment:
Please make this read remote url,  test without url is a issue (see #481).
Goal is to have created nitrate case readable and one can do (with good intentions) `git clone <repo> <custom_name>` and this way `test.node.root` will be `<custom_name>`.
To get an example - instead of {rpms,tests}/component one can prefer to have component/{rpms,tests}. And in this case test.node.root would be 'tests'

Reply:
solved by a3dc8b3fb903a943a646cc220b814eba6bcf02d9